hgtv/templates/index.html.jinja2

```diff
@@ -1,101 +1,66 @@
+    <link rel="apple-touch-icon" href="/static/img/icons/apple-touch-icon-152x152.png">
+    <link rel="mask-icon" href="/static/img/icons/safari-pinned-tab.svg" color="#4DBA87">
+    
+    <meta name="msapplication-TileImage" content="/static/img/icons/msapplication-icon-144x144.png">
+    <meta name="msapplication-TileColor" content="#000000">
+    
+    <link rel="prefetch" href="/static/build/js/0.6a3a87694dd10a1786ef.js" as="script">
+    <link rel="prefetch" href="/static/build/js/1.be76a7ce01c1b9d89297.js" as="script">
+    <link rel="prefetch" href="/static/build/js/2.1400b64d508a07996560.js" as="script">
+    <link rel="prefetch" href="/static/build/js/3.96b6aec0b02ae4ae7d78.js" as="script">
+    <link rel="prefetch" href="/static/build/js/4.ecf90aa50a6ba34f7989.js" as="script">
+    <link rel="prefetch" href="/static/build/js/5.f7c67c73dbd5f291db32.js" as="script">
+    <link rel="prefetch" href="/static/build/js/6.fea783e424faa4bd13b3.js" as="script">
+    <link rel="prefetch" href="/static/build/js/7.e18f38c222e6aae5e370.js" as="script">
+    <link rel="prefetch" href="/static/build/js/8.e8917d614e07f5c520d2.js" as="script">
+    <link rel="prefetch" href="/static/build/js/9.31507849443785922936.js" as="script">
+    <link rel="prefetch" href="/static/build/js/10.80acffb6dbe3318418fa.js" as="script">
+    <link rel="prefetch" href="/static/build/js/11.cf5f5e52f4b44bb42986.js" as="script">
+    <link rel="prefetch" href="/static/build/js/12.6da4fd4f48c404f0d871.js" as="script">
+    <link rel="prefetch" href="/static/build/js/13.d210b8fefe289e1da752.js" as="script">
+    <link rel="prefetch" href="/static/build/js/14.3ecf1c6e950f280bf282.js" as="script">
+    <link rel="preload" href="/static/build/js/vendor.22cbf94a26840c37998c.js" as="script">
+    <link rel="preload" href="/static/build/js/app.996654351593958b7b5f.js" as="script">
+    <link rel="preload" href="/static/build/css/app.a714d73c6c4536062f12a018429f511f.css" as="style">
+    <link rel="prefetch" href="/static/build/js/17.f36783fc58a8ff5d0b52.js" as="script">
+    <link rel="prefetch" href="/static/build/js/18.c4416d6257c867a09d02.js" as="script">
+    <link rel="prefetch" href="/static/build/js/19.2bb50432a5d258868c53.js" as="script">
+    <link rel="prefetch" href="/static/build/js/20.efafc406133266c20682.js" as="script">
+    <link rel="prefetch" href="/static/build/js/21.9907c8433166d21f677b.js" as="script">
+    <link rel="prefetch" href="/static/build/js/22.a8dbe13bf1eb273fd301.js" as="script">
+    <link rel="prefetch" href="/static/build/js/23.fddf6fddab7cf3402cd0.js" as="script">
+    <link rel="prefetch" href="/static/build/js/24.80ffcd06d039039ca7e2.js" as="script">
+    <link rel="prefetch" href="/static/build/js/25.23c398739857e5933cdb.js" as="script">
+    <link rel="prefetch" href="/static/build/js/26.4163a3ed03eade863ab9.js" as="script">
+    <link rel="preload" href="/static/build/js/manifest.6894b1b488c532c4ed20.js" as="script">
```

Comment:
These seems like built files? What happens when they are rebuilt? Have to edit the index file every time?